http/message_store.h

```diff
@@ -0,0 +1,55 @@
+//
+// Created by root on 4/29/24.
+//
+
+#ifndef GOOGOM_MESSAGE_STORE_H
+#define GOOGOM_MESSAGE_STORE_H
+
+
+#include <seastar/core/app-template.hh>
+#include <seastar/core/reactor.hh>
+#include <seastar/core/future.hh>
+#include <seastar/http/httpd.hh>
+#include <seastar/net/api.hh>
+#include <seastar/core/sleep.hh>
+#include <iostream>
+#include <unordered_map>
+#include <optional>
+
+// TODO to be deleted
+// Message Store for REST server
+class message_store {
+public:
+    std::unordered_map<int, std::string> messages;
+    int next_id = 0;
```

Comment:
I know it's a prototype but consider:
1. making it `unsigned int` (I think you don't plan to store the negative values in here because you initialized it at 0 and only increased it later)
2. the int, for now, is large enough, but it might be `unsigned long long` later.